### PR TITLE
Updated MedicationRequest R4 Resource

### PR DESCRIPTION
**Pull request by kckeim**
Had a lot of formatting changes on this one to get it into a similar style to other resources that were recently updated.
Verified good links through document.
Validated that API calls behave as expected with both the expected and unexpected parameters.

Description
----
- Formatting changes throughout, including capitalizing/punctuation, and backticks to flip elements to code.
- Overview paragraph updated slightly.
- Condensed disclaimer to single instance.
- Overhauled Custom Extension section.
  - Linked from the fields list. - Verified that anchor list changes were safe; no other pages in the repository reference this location at all).
  - Re-formatted the common extensions into a table format (like they appear on most other resources).
  - Swedish-specific extensions are still there, and separated out from the common ones.
- Removed `intent` notes that appeared in a few places.
  - All the necessary details are listed alongside details for the `intent` parameter/field anyways.
- Parameter Table + Implementation Notes re-written for clarity and to match standards for our recent resources.
- Added the common "only listed elements are supported" disclaimer for the Create section.
- Updated X-Request-Id to anonymized UUID and added opc-request-id to responses for Create and Patch sections.
*****************************
PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [X] Screenshot(s) of changes attached after changes merged and published.
****************************
Old Overview Paragraph:
image

New:
image

****************************
Added "Extensions" field:
image

Old Extensions Section:
image

New Extensions Section:
image

****************************
Old Parameters + Notes:
image
image

New Parameters + Notes:
image
image

****************************
Old Retrieve:
image

New:
image

****************************
Old Create Headers:
image

New:
image

****************************
Old Responses:
image
image

New Responses with anonymized IDs:
image
image

****************************

### CODE REVIEW

**Review by RicardoCrespo-ora — APPROVED** [review]

**Review by cldeitrich — APPROVED** [review]
[PRODOC-203564](https://jira.cerner.com/browse/PRODOC-203564): Approved for style.

File: content/millennium/r4/clinical/medications/medication-request.md [hunk 1]
```diff
@@ -9,17 +9,18 @@ title: MedicationRequest | R4 API
...
 
-The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+This resource supports the following [HL7 FHIR US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} profiles:
 
 * [US Core MedicationRequest Profile](https://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-medicationrequest.html){:target="_blank"}
 
 The following fields are returned if valued:
 
-* [MedicationRequest id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [MedicationRequest ID](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
```
Comment (amcginn) [inline content/millennium/r4/clinical/medications/medication-request.md:22]: I think we may be missing Identifiers

Comment (amcginn) [reply content/millennium/r4/clinical/medications/medication-request.md:22]: let's just log a defect on this one

**Comment by shushruth-s** [comment]
After Deployment:
Screenshot 2024-02-02 at 3 09 36 PM
Screenshot 2024-02-02 at 3 10 07 PM
Screenshot 2024-02-02 at 3 10 35 PM
Screenshot 2024-02-02 at 3 12 08 PM
Screenshot 2024-02-02 at 3 13 36 PM
Screenshot 2024-02-02 at 3 14 13 PM
Screenshot 2024-02-02 at 3 14 46 PM
Screenshot 2024-02-02 at 3 15 56 PM

**Pull request merged**